analyses/copy_number_consensus_call/Snakefile

```diff
@@ -33,11 +33,11 @@ rule freec_filter:
         ## The first awk looks at column 6 to filter out for loss/gain. Then it prints out 6 of the 7 columns above
```

Comment:
Line 32 comment should be updated to reflect the new output format and this line updated to match.
These comments can also be broken up so the lines are not so wide (100 characters is a good max, but 80-90 is better)

Similarly, update lines 57-58 and 82-83